source/shared/cpp/ObjectModel/BaseInputElement.cpp

```diff
@@ -29,7 +29,10 @@ Json::Value BaseInputElement::SerializeToJsonValue() const
 {
     Json::Value root = BaseCardElement::SerializeToJsonValue();
 
-    root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IsRequired)] = m_isRequired;
+    if (m_isRequired)
+    {
+        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IsRequired)] = m_isRequired;
+    }
```

Comment:
This change will require updates to the UWP test app and unit tests to update the expected behavior.

Reply:
Also, btw, i have this locally as part of my InputValidation "prototyping" so if we don't need this for 1.2.1 we can drop this change and wait for that to come along.

---
In reply to: [300811694](https://github.com/microsoft/AdaptiveCards/pull/3131#discussion_r300811694) [](ancestors = 300811694)